comgr/PKGBUILD

```diff
@@ -2,22 +2,22 @@
 
 pkgname=comgr
 pkgdesc='Radeon Open Compute - compiler support'
-pkgver=3.3.0
-pkgrel=4
+pkgver=3.5.0
+pkgrel=1
 arch=('x86_64')
 url='https://github.com/RadeonOpenCompute/ROCm-CompilerSupport'
 license=('custom:NCSAOSL')
 depends=(z3 zlib)
 makedepends=(cmake rocm-cmake rocm-device-libs llvm-amdgpu)
-source=("$url/archive/rocm-$pkgver.tar.gz"
-        "comgr-find-lld-includes.patch")
-sha256sums=('01e2524e0f28ecd6f46c9720f279207de935d826b0172158792aa3ec86af9ca7'
-            '4571b16961f15249e8cc8b9a9ae7f0863600345aa5e95959192149eacdb01d2e')
+source=("${pkgname}-${pkgver}.tar.gz::$url/archive/rocm-$pkgver.tar.gz")
+        # 'comgr-find-lld-includes.patch')
```

Comment:
Let's just go ahead and delete this patch file and anything to do with the patch, if it doesn't work anymore.

Reply:
Done.